requirements/skills-desktop.txt

```diff
@@ -0,0 +1,3 @@
+# TODO - publish
+# https://github.com/OpenVoiceOS/skill-ovos-dictation
```

Comment:
Why is this a desktop skill?  Could it not be used on a headless device?

Reply:
This is a good point - the skill runtime_requirements do specify it doesn't require a GUI

Reply:
I still don't think this should be a desktop skill unless I am missing something.

Reply:
its commented out because its not published, needs automations in the repo

ill merge for now until that is done as it should not block 0.0.8